frontend/src/streamDetails/__tests__/StreamDetailsWrapper.test.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { Box, Container, Grid } from "@material-ui/core";
+import { beforeEach, describe, expect, it } from "@jest/globals";
+
+import StreamDetailsWrapper from "../StreamDetailsWrapper";
+import DashboardCard from "../../general/dashboard/DashboardCard";
+import StreamDeviceDetails from "../StreamDeviceDetails";
+import DeleteStreamDialogOpener from "../DeleteStreamDialogOpener";
+
+import DeviceInfo from "../../model/DeviceInfo";
+import StreamInfo from "../../model/StreamInfo";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<StreamDetailsWrapper/> functional component", () => {
+  let wrapper;
+
+  describe("returns a component that", () => {
+    const dummySender = new DeviceInfo(1, 1, 1, 1, 1, 1, [1, 2]);
+    const dummyReceiver = new DeviceInfo(2, 2, 2, 2, 2, 2, [3, 4]);
+    const dummyStream = new StreamInfo(1, dummySender, dummyReceiver, 2, 3);
+    beforeEach(() => {
+      wrapper = Enzyme.shallow(<StreamDetailsWrapper stream={dummyStream} />);
+    });
+    it("Contains 1 Container component with expected props", () => {
+      expect(wrapper.find(Container)).toHaveLength(1);
+    });
+    it("Contains 5 Grid components with expected props", () => {
+      const gridComponents = wrapper.find(Grid);
+
+      expect(gridComponents).toHaveLength(5);
+      // iterate through grid components to test each
+
+      const containerGridProps = gridComponents.at(0).props();
+      expect(containerGridProps.container).toBeTruthy();
+      expect(containerGridProps.spacing).toBe(3);
+
+      let itemGridProps = gridComponents.at(1).props();
+      expect(itemGridProps.item).toBeTruthy();
+      expect(itemGridProps.xs).toBe(6);
+
+      itemGridProps = gridComponents.at(2).props();
+      expect(itemGridProps.item).toBeTruthy();
```

Comment:
why are you checking for truthy? it should return true

Reply:
7b8fa32